server/src/test/java/org/elasticsearch/index/fielddata/ScriptDocValuesLongsTests.java

```diff
@@ -135,12 +128,12 @@ public Void run() {
             }
         }, noPermissionsAcc);
 
-        assertThat(warnings, containsInAnyOrder(
+        assertThat(warnings, hasItems(
```

Comment:
This seems to be making the check lenient, so other warnings could also exist. I think this test should be verify all warnings that are emitted?